src/point_one/fusion_engine/messages/solution.h

```diff
@@ -323,7 +323,16 @@ struct alignas(4) SatelliteInfo {
  */
 enum class CalibrationStage : uint8_t {
   UNKNOWN = 0, ///< Calibration stage not known.
-  MOUNTING_ANGLE = 1, ///< Estimating IMU mounting angles.
+  /**
+   * Performing initial estimation of IMU mounting angles. Reported position
+   * solution may have reduced accuracy.
+   */
+  MOUNTING_ANGLE_INITIAL_CONVERGENCE = 20,
```

Comment:
It's unclear to me to if these are the start or end of these stages. Also perhaps the reasons are too directly related to what's happening under the hood, originally we talked.aboit just having a coarse calibration done generically...

Reply:
The stage indicates "this thing is ongoing", and you'd normally be in that stage for some time until moving to the next one.

Normally most of the progression is an internal detail that isn't too important. We figured this one distinction might be useful since the performance may change notably when we get to final convergence, and someone may want to use that information. But we could also skip this for now?

Reply:
Shall we make no change at all in this regard for now?

Reply:
Under fusion-engine-client.